blackjack.py

```diff
@@ -79,23 +79,32 @@ def print_entire_hand(self):
         for card in self.hand:
             print card
 
+    def check_natural_win(self):
+        if len(self.hand) == 2:
+            if self.hand[0].value == 'Ace' and self.hand[1].value in ['Jack', 'Queen', 'King']:
+                return True
+            elif self.hand[0].value in ['Jack', 'Queen', 'King'] and self.hand[1].value == 'Ace':
+                return True
+            else:
+                return False
+        return False
```

Comment:
The multiple `return True` and `return False` are a bit of a code smell.
1. If you have a function like so:
```
def f(x):
    if x == 0:
        return True
    elif x == 1:
        return True
    else:
        return False
```

You should refactor to get this:
```
def f(x):
    if x == 0 or x == 1:
        return True
    else:
        return False
```

Or even more pythonic:
```
def f(x):
    if x in (0, 1):
        return True
    else:
        return False
```

2. Also, you rarely see `return True` or `return False` because you can directly return your conditions. For example, instead of the function `f` above, you should write:
```
def f(x):
    return x in (0, 1)
```

In this particular case it's not that easy because of the length of those conditions, but there may be a way.

Reply:
Yeah I feel uneasy when I look at the amount of True/False statements in that function. 
Let me see if I can find a better way